### ISSUE TITLE
Gravity X / Gravity Y interface isn't updated on loading

### ISSUE THREAD

**Issue by phantogast** [pierredavidbelanger/apparticle#12]
Hi Pierre! I love Apparticle, I've been using it a lot lately, thank you very much for making it! It's a pleasure to work with! :D

I have however noticed that when I load a .plist file the Gravity X & Gravity Y parameters don't update. The particles display correctly using the setting in the plist, but the interface always displays 0 instead of the values in use.

Thanks again for building Apparticle, I like it a lot! :D

**Comment by pierredavidbelanger** [comment]
Thank you for you feedback!

I will have a look at this.

What version did you use ? 
Did you get it from the AppStore ?

**Comment by pierredavidbelanger** [comment]
You are absolutely right, those values are correct visually, but are at 0 in the settings.
